### PR TITLE
Add .npmignore

### PR DESCRIPTION
**Pull request by nkbt**
There are some files that should not be included in npm package.

## BEFORE
```sh
[nkbt@nkbt] impress $ npm install ~/github/impress.js -s
/private/tmp/impress
└── impress.js@0.5.3

[nkbt@nkbt] impress $ find .
.
./node_modules
./node_modules/impress.js
./node_modules/impress.js/.github
./node_modules/impress.js/.github/CONTRIBUTING.md
./node_modules/impress.js/.github/ISSUE_TEMPLATE.md
./node_modules/impress.js/.github/PULL_REQUEST_TEMPLATE.md
./node_modules/impress.js/.jshintrc
./node_modules/impress.js/.npmignore
./node_modules/impress.js/apple-touch-icon.png
./node_modules/impress.js/bower.json
./node_modules/impress.js/CHANGELOG.md
./node_modules/impress.js/css
./node_modules/impress.js/css/impress-demo.css
./node_modules/impress.js/favicon.png
./node_modules/impress.js/index.html
./node_modules/impress.js/js
./node_modules/impress.js/js/impress.js
./node_modules/impress.js/karma.conf.js
./node_modules/impress.js/LICENSE
./node_modules/impress.js/package.json
./node_modules/impress.js/README.md
./node_modules/impress.js/test
./node_modules/impress.js/test/bootstrap.js
./node_modules/impress.js/test/core_tests.js
```

## AFTER:
```sh
[nkbt@nkbt] impress $ npm install ~/github/impress.js -s
/private/tmp/impress
└── impress.js@0.5.3

[nkbt@nkbt] impress $ find .
.
./node_modules
./node_modules/impress.js
./node_modules/impress.js/apple-touch-icon.png
./node_modules/impress.js/CHANGELOG.md
./node_modules/impress.js/css
./node_modules/impress.js/css/impress-demo.css
./node_modules/impress.js/favicon.png
./node_modules/impress.js/index.html
./node_modules/impress.js/js
./node_modules/impress.js/js/impress.js
./node_modules/impress.js/karma.conf.js
./node_modules/impress.js/LICENSE
./node_modules/impress.js/package.json
```

**Make sure these boxes are checked before submitting your Pull Request, thank you!**

- [x] I have read and complied with the [Pull Request Contributing Guidelines](https://github.com/impress/impress.js/blob/master/CONTRIBUTING.md#pull-requests).
- [x] I checked if an [an issue](https://github.com/impress/impress.js/issues) is necessary for my Pull Request and made sure it exists.
- [x] I have created a [topic branch](https://github.com/dchelimsky/rspec/wiki/Topic-Branches).
- [x] I have used the [Search Tool](https://github.com/impress/impress.js/search?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen&type=Issues) to make sure there are no open Pull Requests fixing the same thing.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/.*/
+/.*
```
Comment (FagnerMartinsBrack) [inline .npmignore]: If I get it right, the first pattern specify all **root directories** that starts with a dot `.`, and the second pattern specifies all **root files** that starts with a dot `.`?

Shouldn't we also leave out `karma.conf.js`?

Comment (nkbt) [inline .npmignore]: Right. Karma too. I'll update.

**Comment by nkbt** [comment]
Added karma.conf.js

**Comment by nkbt** [comment]
PS: build is failing since it has no circleci config yet.

**Comment by FagnerMartinsBrack** [comment]
@nkbt Want to rebase on master? It seems [CircleCI is working](https://github.com/impress/impress.js/pull/571), yay!

**Comment by nkbt** [comment]
Rebased, added `circle.yml`

**Comment by nkbt** [comment]
\o/
I have some more to come. Stay tuned

**Pull request closed without merging**